portality/scripts/remove_heloise_policy.py

```diff
@@ -0,0 +1,38 @@
+import csv
```

Comment:
Suggest that this script lives in the `migration` directory, as it's a one-time affair for a specific change.  Then the resources file related to it can be wrapped up in that directory as well, and they won't sit in the `scripts` directory never being used again